intera_examples/scripts/go_to_cartesian_pose.py

```diff
@@ -0,0 +1,179 @@
+#! /usr/bin/env python
+
+# Copyright (c) 2016-2017, Rethink Robotics Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rospy
+import argparse
+from motion_interface import (
+    MotionTrajectory,
+    MotionWaypoint,
+    MotionWaypointOptions
+)
+from motion_msgs.msg import TrajectoryOptions
+from geometry_msgs.msg import PoseStamped
+import PyKDL
```

Comment:
This is the first link I found for installing PyKDL:

http://sdk.rethinkrobotics.com/wiki/Baxter_PyKDL